package.json

```diff
@@ -9,13 +9,15 @@
     "undoseeds": "babel-node node_modules/.bin/sequelize db:seed:undo",
     "migration": "babel-node node_modules/.bin/sequelize db:migrate",
     "undomigration": "export NODE_ENV=test && npm run undoseeds && babel-node node_modules/.bin/sequelize db:migrate:undo:all",
-    "test": "export NODE_ENV=test && npm run undomigration && npm run migration && npm run seeds && coverage mocha ./test --recurssive --no-timeout --exit --require @babel/register",
     "dev": "npm run migration && npm run seeds && nodemon --exec babel-node ./src/app.js",
-    "generate-lcov": "nyc report --reporter=text-lcov > lcov.info",
-    "coveralls-coverage": "coveralls < lcov.info",
-    "coveralls": "npm run generate-lcov && npm run coveralls-coverage"
+    "test": "export NODE_ENV=test && npm run undomigration && npm run migration && npm run seeds && nyc mocha ./test --recurssive --no-timeout --exit --require @babel/register",
+    "coveralls": "nyc report --reporter=text-lcov | coveralls"
   },
   "author": "Andela Simulations Programme",
+  "repository": {
```

Comment:
This is not what i posted in https://github.com/lemurheavy/coveralls-public/issues/208#issue-27138842